source/scripting/VehicleDoor.cs

```diff
@@ -75,11 +75,11 @@ public void Close(bool instantly)
 		}
 		public void BreakOff()
 		{
-			Function.Call(Hash.SET_VEHICLE_DOOR_BROKEN, _owner.Handle, (int)Index, true);
+			BreakOff(true);
 		}
-		public void Delete()
+		public void BreakOff(bool stayInTheWorld)
```

Comment:
You may use default parameters instead here:
```
public void BreakOff(bool stayInTheWorld = true)
```

By the way, I'd prefer a `CanBeBroken` and `Break()` over the current names.